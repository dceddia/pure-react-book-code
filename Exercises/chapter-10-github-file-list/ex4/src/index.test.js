import ReactDOM from 'react-dom';
jest.mock('react-dom', () => ({ render: jest.fn() }));
global.document.querySelector = () => {};

import { Board, board } from './index';
import React from 'react';
import renderer from 'react-test-renderer';

it('renders', () => {
  const tree = renderer.create(<Board board={board} />).toJSON();
  expect(tree).toMatchSnapshot();
});
