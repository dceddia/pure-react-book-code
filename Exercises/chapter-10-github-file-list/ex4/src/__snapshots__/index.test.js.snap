// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<div
  className="board"
>
  <h4
    className="board-title"
  >
    A Demo
  </h4>
  <div
    className="board-lists"
  >
    <div
      className="list"
    >
      <h4
        className="list-title"
      >
        Phone Features
      </h4>
      <ul
        className="list-items"
      >
        <li
          className="list-item"
        >
          Subwoofer
        </li>
        <li
          className="list-item"
        >
          Non-porous, washable
        </li>
        <li
          className="list-item"
        >
          Wings
        </li>
        <li
          className="list-item"
        >
          Beveled Bezel
        </li>
        <li
          className="list-item"
        >
          Bezeled Bevel
        </li>
        <li
          className="list-item"
        >
          Seedless
        </li>
      </ul>
      <div
        className="add-card"
      >
        Add a card...
      </div>
    </div>
    <div
      className="list"
    >
      <h4
        className="list-title"
      >
        Lyrics
      </h4>
      <ul
        className="list-items"
      >
        <li
          className="list-item"
        >
          This was a triumph
        </li>
        <li
          className="list-item"
        >
          I'm making a note here
        </li>
        <li
          className="list-item"
        >
          HUGE SUCCESS
        </li>
        <li
          className="list-item"
        >
          It's hard to overstate my satisfaction
        </li>
      </ul>
      <div
        className="add-card"
      >
        Add a card...
      </div>
    </div>
  </div>
</div>
`;
